docs/plugins/gatsby-theme-flow/src/components/flow-status/components/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+
+import { getNextSporkDate, getNetworkVersion } from "../helpers";
+
+import RecentPost from "./recentPost";
+import StatusCard from "./statusCard";
+
+function NetworkVersion(networkName) {
+  const [version, setVersion] = useState("");
+  useEffect(() => {
+    async function get() {
+      try {
+        const v = await getNetworkVersion(networkName);
+        setVersion(v);
+      } catch (e) {
+        console.log("Error getting network version...");
+      }
+    }
+    get();
+  }, []);
+  return (
+    <>
+      <h4>Version</h4>9
```

Comment:
We should figure out how to fetch these dynamically. The component here is ready to make an async call for this information, but isn't used currently. The values for next spork date and spork version have to be added manually for now.

Reply:
Good to know 👍

Reply:
For now can these be props passed into the components? I think it might be a while until there's a dynamic source of truth for this. 

You could even lean on a bit of composition/mdx to make it easier for future maintainers to see how the content is updated.

Something like this:

```mdx
// status.mdx

## Chains

<NetworkVersion 
  network="Mainnet" 
  version="9"
  versionUrl="LINK_TO_RELEASE_NOTES"
  nextSporkDate="June 23, 2021" />

<NetworkVersion 
  network="Testnet" 
  version="9"
  versionUrl="LINK_TO_RELEASE_NOTES"
  nextSporkDate="June 23, 2021" />
 
<NetworkVersion 
  network="Canarynet" 
  version="9"
  versionUrl="LINK_TO_RELEASE_NOTES"
  nextSporkDate="June 23, 2021" />
  
## Announcements

<AnnouncementsWrapper />
```

Reply:
This also allows the networks to have different versions and spork dates.

Reply:
Nice. yeah that looks clean. 
I've updated the PR so we can include the version information via the markdown file for easy editing.